### DISCUSSION TITLE
how to put last column to most right

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [TanStack/table#5211] [Q&A]
![image](https://github.com/TanStack/table/assets/35369563/7fb77243-d563-407a-b01e-5992b37bc070)
how to put last column to most right I don't want any gap. I want all to strect and fit

**Comment by maliiiith** [comment]
Hi,

You may have already found the solution to this. But since this question remains unanswered, I'll leave an answer to anyone else come looking.

You can set `display: flex; justify-content: end;` on td and th of the column you want to align far right.

Also, if you are working with Typescript, https://github.com/TanStack/table/discussions/4824#discussioncomment-5748483 answer will help to extend `ColumnMeta`, so you can pass classes directly from the column definition.

**Reply by 0xMALVEE** [reply]
after 2 years. 😒
